doc/source/examples/unresolved-cfd-dem/liquid-solid-fluidized-bed/liquid-solid-fluidized-bed.rst

```diff
@@ -0,0 +1,500 @@
+
+To set-up the cylinder fluidized bed case, we first fill the bed with particles.
+
+We first introduce the different sections of the parameter file ``dem-packing-in-lsfb.prm`` needed to run this simulation.
+
+The parameter is divided into `subsections`, presented individually as follows.
+
+Mesh
+~~~~~
+
+In this example, we are simulating a cylindrical fluidized bed that has a half length of 0.55 m (10 cm higher than the fluidization region in the experimental setup), and a diameter of 10 cm. We use the `subdivided_cylinder GridGenerator <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html#a95f6e6a7ae2fe3a862df035dd2cb4467:~:text=%E2%97%86-,subdivided_cylinder,-()>`_  in order to generate the mesh. The square bed is divided 132 times in the x direction (height) and 12 times in y and z directions (6 times in the radius). The following portion of the DEM parameter file shows the function called:
+
+.. code-block:: text
+
+    subsection mesh
+      set type               = dealii
+      set grid type          = subdivided_cylinder
+      set grid arguments     = 33:0.05:0.55
+      set initial refinement = 2
+      set expand particle-wall contact search = true
+    end
+
+.. note::
+    Note that, since the mesh is cylindrical, ``set expand particle-wall contact search = true``. Details in the `DEM mesh parameters guide <../../../parameters/dem/mesh.html>`_.
+
+A cross section of the resulting mesh is presented in the following figure.
+
+.. figure:: images/mesh_cross_sec.png
+    :alt: Mesh cross-section.
+    :align: center
+
+    Cross-section of the mesh used in liquid-solid fluidized bed simulations.
+
+Floating Walls
+~~~~~~~~~~~~~~~~~~~
+
+A floating wall is added 10 cm above the bottom of the mesh, so that void fraction discontinuities can be avoided. The remaining region above the floating wall is 1 m high, as in the experimental setup.
+
+.. code-block:: text
+
+    subsection floating walls
+      set number of floating walls = 1
+      subsection wall 0
+        subsection point on wall
+          set x = -0.45
+          set y = 0
+          set z = 0
+        end
+      subsection normal vector
+        set nx = 1
+        set ny = 0
+        set nz = 0
+      end
+      set start time = 0
+      set end time   = 50
+      end
+    end
+
+.. note::
+    Note that ``end time`` is higher than ``time end`` in ``simulation control``, so that the floating wall does not disappear in the middle of the simulation.
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here, we define the time-step and the simulation end time.
+
+.. code-block:: text
+
+    subsection simulation control
+      set time step        = 0.000005
+      set time end         = 10.5
+      set log frequency    = 1000
+      set output frequency = 20000
+      set output path      = ./output_dem/
+    end
+
+It is important to define ``time end`` according to particles insertion (explained further in this example).
```

Comment:
This is very vague. I would say this it is important to define the time end to include also the time required to insert the particles in the first simulation